docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -454,26 +454,25 @@ spec:
   [etcd best practice](https://etcd.io/docs/v3.3.12/faq/#why-an-odd-number-of-cluster-members).
 - However, allow a control plane using an external etcd cluster to scale up to other numbers such as 2 or 4.
 - Scale up operations must not be done in conjunction with an upgrade operation, this should be enforced using a validation webhook.
-- Scale up operations *could* be blocked based on the Health status
-  - for stacked etcd quorum must be considered
-  - for external etcd only availability of etcd should be considered.
-  - Health status is described below.
+- Scale up operations are blocked based on Etcd and control plane health checks.
+  - See [Healthchecks](#Healthchecks) below.
 
 ![controlplane-init-6](images/controlplane/controlplane-init-6.png)
 
 ##### Scale Down
 
 - Scale down a stacked control plane
-  - For each replica being deleted:
-    - Remove the replica’s etcd member from the etcd cluster
-    - Update the kubeadm generated config map
+  - Wait for deletes to finish before scaling down
+  - Find the oldest machine, or machine marked for removal and target that as the replica to remove
+  - Apply a label to the machine marked for removal (allows for reentrancy)
+  - Remove the replica’s etcd member from the etcd cluster
+  - Update the kubeadm generated config map
```

Comment:
We will also be adding annotations when the etcd member removal and kubeadm config modification steps are done.